src/modules/UI/scenes/CryptoExchange/CryptoExchangeSceneComponent.js

```diff
@@ -160,7 +147,7 @@ export class CryptoExchangeSceneComponent extends Component<Props, State> {
             keyboardShouldPersistTaps={Constants.ALWAYS}
             contentContainerStyle={style.scrollViewContentContainer}
           >
-            <CryptoExchangeConnector style={style.exchangeRateBanner} />
+            <CryptoExchangeMessageConnector style={style.exchangeRateBanner} />
```

Comment:
agreed, see comment where we discussed this earlier.. I would have done wired but this was really a rename and slight adjustments to variable names in an existing file. Since it was no longer used for the exchange rate and error messages I wanted to have names that better reflected what the actual component was.